docs/rackhd/static_file_server.rst

```diff
@@ -0,0 +1,94 @@
+.. _static-file-server-label:
+
+Static File Service Setup
+---------------------------
+.. _on-http: https://github.com/RackHD/on-http
+.. _nginx: https://www.nginx.com/
+.. _install nginx: https://www.nginx.com/resources/wiki/start/topics/tutorials/install/
+.. _nginx_conf: https://www.nginx.com/resources/wiki/start/topics/examples/full/
+.. _config.json: https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json
+.. _httpProxies: http://rackhd.readthedocs.io/en/latest/rackhd/configuration.html?highlight=httpProxies
+.. _httpStaticRoot: http://rackhd.readthedocs.io/en/latest/rackhd/configuration.html?highlight=httpStaticRoot
+
+
+There are two kinds of static files in RackHD, and this section introduces a mechanism to move one type of
```

Comment:
Might be good to give a very short introduction to those two types of static files, or pointing user to a link that introduces those two types.